daml-model/daml/Tests/LoanManagementTest.daml

```diff
@@ -30,7 +30,7 @@ given_balance_below_threshold_then_create_loan = do
     aliceBalanceCid <- submit bank11 do
         createCmd Balance with
             provider = bank11
-            owner = Some users.alice
+            owner = BalanceOwner with name = "alice", party = Some users.alice
```

Comment:
What about using `balanceOwner`?

Reply:
Done. Moved `balanceOwner` function to separate `Util` module